### PR TITLE
Support hashtags

### PR DESCRIPTION
**Pull request by xerZV**
Need to be in a separate branch, not in the `master`.
But I am not allowed to create a new branch.

### CODE REVIEW

**Comment by alexal1** [comment]
> But I am not allowed to create a new branch

I didn't know that I have to add you as a collaborator first, my bad. Fixed now.

File: insomniac.py [hunk 1]
```diff
@@ -143,6 +203,18 @@ def _parse_arguments():
     return True, args
 
 
+def check_hashtags(hashtags):
+    print("Checking the hashtags")
+
+    for i in range(len(hashtags)):
+        hashtag = hashtags[i]
+        if hashtag != "#":
```
Comment (alexal1) [inline insomniac.py:211]: I cannot understand what this function actually does

File: utils.py [hunk 2]
```diff
@@ -13,6 +14,11 @@
 COLOR_UNDERLINE = '\033[4m'
 
 
+class Mode(Enum):
```
Comment (alexal1) [inline utils.py:17]: I consider `utils.py` as a place for simple utility functions, but this seems more like a part of core logic. Better place it in `insomniac.py`

**Comment by GabrieleAnsaldo** [comment]
Hey @xerZV , did you manage to find a way to open the first picture in Recent?

**Comment by alexal1** [comment]
This PR seems to be abandoned, let's concentrate on #49 instead.

**Pull request closed without merging**